back/app/jobs/delete_invites_job.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Also deletes users associated with expired && pending invites (accepted_at: nil)!
+class DeleteInvitesJob < ApplicationJob
+  self.priority = 90 # pretty low priority (lowest is 100)
+
+  # @param [expiry_time,ActiveSupport::TimeWithZone] the expiry time limit for invites
+  def run(expiry_time = Invite::EXPIRY_DAYS.days.ago)
+    invites_to_destroy = Invite.where('created_at < ?', expiry_time)
+    payload = { destroyed_invites: [] }
+
+    invites_to_destroy.each do |invite|
+      invite.destroy!
+      payload[:destroyed_invites] << invite.as_json
+    rescue StandardError => e
+      ErrorReporter.report e
+    end
```

Comment:
Iterating, rather than batch destroying, to facilitate specific error logging + accurate record in payload.